src/excmds.ts

```diff
@@ -2931,6 +2931,13 @@ export function bind(...args: string[]) {
     let args_obj = parse_bind_args(...args)
     let p = Promise.resolve()
     if (args_obj.excmd != "") {
+        for (let i = 0; i < args_obj.key.length; i++) {  // Check if any initial subsequence of the key exists and will shadow the new binding
+            let key_sub = args_obj.key.slice(0, i);
+            if (config.get(args_obj.configName, key_sub)) {
+                fillcmdline_notrail("# Warning: bind `"+key_sub+"` exists and will shadow `" + args_obj.key + "`. Try running `:unbind "+key_sub+"` first");
```

Comment:
My reading of this is that we still allow the bind to be set. I think this message could make that clearer. I think just deleting "first" from the end would help.